iosApp/Survey/Sources/Presentation/Views/FadePaginationView.swift

```diff
@@ -28,47 +28,51 @@ struct FadePaginationView<T>: View {
 
     var body: some View {
         ZStack {
-            nextView(items[nextPage])
-            currentView(items[currentPage])
-                .opacity(currentVisibility)
-                .gesture(
-                    DragGesture(minimumDistance: minimumTurnDistance, coordinateSpace: .local)
-                        .onChanged { value in
-                            switch value.translation.width {
-                            case ...0: nextPage = min(items.count - 1, currentPage + 1)
-                            case 0...: nextPage = max(0, currentPage - 1)
-                            default: return
+            if items.count > nextPage {
```

Comment:
@blyscuit Can you double-check this one? The code changes are the same with #102

Reply:
Just noticed! You have conflicts on this PR :D

Reply:
Thank you, I just rebased #102 for the comments. Will rebase this one now.